index.html

```diff
@@ -229,6 +207,27 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="#" method="post" class="form-contact-me">
```

Comment:
- [ ] Please use [Formspree](https://formspree.io/html) in your form to collect user data. 
  - To use formspree, Create a formspree account and place `action="https://formspree.io/f/xxxxxx"` in your form action attribute, Replace `xxxxx` with your formspree code.